Dockerfile

```diff
@@ -13,19 +13,20 @@ RUN curl -L https://archive.apache.org/dist/pinot/apache-pinot-$PINOT_VERSION/ap
     mv apache-pinot-$PINOT_VERSION-bin $PINOT_HOME && \
     rm -rf $PINOT_HOME/examples && \
     rm -rf $PINOT_HOME/lib/* && \
-    rm -rf $PINOT_HOME/plugins/*
+    rm -rf $PINOT_HOME/plugins/* && \
+    rm -rf $PINOT_HOME/plugins-external/*
 
 # Fetch jar
 RUN curl -L -o $PINOT_HOME/lib/pinot-all-${JITPACK_TAG}-shaded.jar \
         https://jitpack.io/com/github/${JITPACK_REPO}/pinot-distribution/${JITPACK_TAG}/pinot-distribution-${JITPACK_TAG}-shaded.jar
 
 # Fetch plugin jars
-RUN for artifactId in pinot-kafka-2.0 pinot-kinesis pinot-thrift pinot-json pinot-parquet pinot-orc pinot-csv pinot-confluent-avro pinot-avro pinot-protobuf pinot-batch-ingestion-standalone pinot-batch-ingestion-hadoop pinot-batch-ingestion-spark pinot-hdfs pinot-adls pinot-gcs pinot-s3 pinot-minion-builtin-tasks pinot-segment-uploader-default pinot-segment-writer-file-based pinot-dropwizard pinot-yammer; do \
```

Comment:
Removed the following 3 plugins having critical vulnerabilities:
1. pinot-input-format/pinot-orc
2. pinot-input-format/pinot-parquet
3. pinot-batch-ingestion/pinot-batch-ingestion-spark

Reply:
Assuming we are not using them?